packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -75,16 +88,45 @@ class PlatformClusterManager {
   final String identifier;
 }
 
+/// Pigeon equivalent of the Offset class
+class PlatformOffset {
+  PlatformOffset(this.dx, this.dy);
+
+  final double dx;
+  final double dy;
+}
+
 /// Pigeon equivalent of the Marker class.
 class PlatformMarker {
-  PlatformMarker(this.json);
+  PlatformMarker({
+    required this.markerId,
+    this.alpha = 1.0,
+    required this.anchor,
+    this.consumeTapEvents = false,
+    this.draggable = false,
+    this.flat = false,
+    this.icon = const <Object>['defaultMarker'],
+    this.infoWindow = const <String?, Object>{'anchor': <Object>[0.5, 0.0]},
+    required this.position,
+    this.rotation = 0.0,
+    this.visible = true,
+    this.zIndex = 0.0,
+    this.clusterManagerId,
```

Comment:
Same note here.

Reply:
Should there be a note here? (my guess is no since id is a string but thought it was worth asking.